### PR TITLE
Method is register

### PR DESCRIPTION
**Pull request by gpibarra**

### CODE REVIEW

**Comment by TheMisir** [comment]
@mehdok please approve and push to pub.dev as soon as possible, I need this feature in production 😟

File: android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt [hunk 1]
```diff
@@ -151,6 +152,20 @@ class BackgroundLocatorPlugin()
             stopIsolateService(context)
         }
 
+        @JvmStatic
+        private fun isRegisterLocator(context: Context,
+                                  client: FusedLocationProviderClient,
+                                  result: Result?) {
+            if (IsolateHolderService?.isRunning) {
+                Log.d("BackgroundLocatorPlugin", "Check Locator service: running")
```
Comment (mehdok) [inline android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt]: Please remove log.

File: android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt [hunk 2]
```diff
@@ -151,6 +152,20 @@ class BackgroundLocatorPlugin()
...
+                result?.success(true)
+            } else {
+                Log.d("BackgroundLocatorPlugin", "Check Locator service: not running")
```
Comment (mehdok) [inline android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt]: Please remove log.

File: android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt [hunk 3]
```diff
@@ -151,6 +152,20 @@ class BackgroundLocatorPlugin()
             stopIsolateService(context)
         }
 
+        @JvmStatic
+        private fun isRegisterLocator(context: Context,
+                                  client: FusedLocationProviderClient,
```
Comment (mehdok) [inline android/src/main/kotlin/rekab/app/background_locator/BackgroundLocatorPlugin.kt]: There is no need for `client`

**Pull request merged**